docs/api-v1.json

```diff
@@ -0,0 +1,1136 @@
+      "description": "Listing operations used for managing listings."
+    },
+    {
+      "name": "default",
+      "description": "Default operations used for handling unknown requests."
+    }
+  ],
+  "paths": {
+    "/": {
+      "get": {
+        "summary": "ping the server",
+        "description": "Returns a simple message to verify that the server is up and running.<br><h3>Permitted Roles</h3><ul><li>ANONYMOUS</li><li>LANDLORD</li><li>STUDENT</li><li>TEAM</li><li>ADMIN</li></ul>",
+        "operationId": "ping",
+        "security": [
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "string",
+                  "default": "The API is running."
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v1/auth/register": {
+      "post": {
+        "summary": "register a new user",
+        "description": "registers and logs in a new user (token provided in the response)<br><h3>Permitted Roles</h3><ul><li>ANONYMOUS</li><li>LANDLORD</li><li>STUDENT</li><li>TEAM</li><li>ADMIN</li></ul>",
+        "operationId": "register",
+        "tags": [
+          "authentication"
+        ],
+        "security": [
+        ],
+        "requestBody": {
+          "$ref": "#/components/requestBodies/User"
+        },
+        "responses": {
+          "200": {
+            "$ref": "#/components/responses/UserGet"
+          },
+          "400": {
+            "$ref": "#/components/responses/BadRequestError"
+          },
+          "409": {
+            "$ref": "#/components/responses/ConflictError"
+          },
+          "500": {
+            "$ref": "#/components/responses/InternalServerError"
+          },
+          "default": {
+            "$ref": "#/components/responses/UndefinedError"
+          }
+        }
+      }
+    },
+    "/v1/auth/login": {
+      "put": {
+        "summary": "log in a user",
+        "description": "used to log in as an already registered user (token is provided in response)<br><h3>Permitted Roles</h3><ul><li>ANONYMOUS</li><li>LANDLORD</li><li>STUDENT</li><li>TEAM</li><li>ADMIN</li></ul>",
+        "operationId": "login",
+        "tags": [
+          "authentication"
+        ],
+        "requestBody": {
+          "$ref": "#/components/requestBodies/Authentication"
+        },
+        "security": [
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/components/responses/UserGet"
+          },
+          "400": {
+            "$ref": "#/components/responses/BadRequestError"
+          },
+          "500": {
+            "$ref": "#/components/responses/InternalServerError"
+          },
+          "default": {
+            "$ref": "#/components/responses/UndefinedError"
+          }
+        }
+      }
+    },
+    "/v1/users/": {
+      "get": {
```

Comment:
/v1/users

Reply:
both works, so it actually doesn't matter